lib/resources/r4/questionnaire_response.yaml

```diff
@@ -2,6 +2,19 @@
 name: QuestionnaireResponse
 field_name_base_url: https://hl7.org/implement/standards/fhir/R4/questionnaireresponse-definitions.html#QuestionnaireResponse
 fields:
+  - name: Answer Value
+    url: https://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.answer.value[x]
+    terminology_name: item.item.answer.value
+    type: Coding
+    description: The value of one of the permitted answers for a "choice-answer" question.
```

Comment:
An answer from a list of available Coding answer options.

Im trying to somewhat match the choice-answer structure def which reads: Indicates that answers are expected to come from a list of available Coding answer options.

But again im not sure

Reply:
No I like that: b35ce2920cb3e4b121f963ee25017323f5435386